bitnami/opensearch/values.yaml

```diff
@@ -0,0 +1,2342 @@
+  ## @param master.pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
+  ##
+  pdb:
+    create: false
+    minAvailable: 1
+    maxUnavailable: ""
+
+  ## @param master.nameOverride String to partially override opensearch.master.fullname
+  ##
+  nameOverride: ""
+  ## @param master.fullnameOverride String to fully override opensearch.master.fullname
+  ##
+  fullnameOverride: ""
+  ## @param master.servicenameOverride String to fully override opensearch.master.servicename
+  ##
+  servicenameOverride: ""
+  ## @param master.annotations [object] Annotations for the master statefulset
+  ##
+  annotations: {}
+  ## @param master.updateStrategy.type Master-elegible nodes statefulset stategy type
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+  ##
+  updateStrategy:
+    type: RollingUpdate
+  ## Opensearch resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param master.resources.limits The resources limits for opensearch containers
+  ## @param master.resources.requests The requested resources for opensearch containers
+  ##
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    cpu: 250m
+    ##    memory: 256Mi
+    requests: {}
+  ## @param master.heapSize Opensearch master-eligible node heap size.
+  ## Note: The recommended heapSize is half of the container's memory.
+  ## If omitted, it will be automatically set.
+  ## Example:
+  ##   heapSize: 128m
+  ##
+  heapSize: 128m
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param master.podSecurityContext.enabled Enabled master-elegible pods' Security Context
+  ## @param master.podSecurityContext.fsGroup Set master-elegible pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param master.containerSecurityContext.enabled Enabled master-elegible containers' Security Context
+  ## @param master.containerSecurityContext.runAsUser Set master-elegible containers' Security Context runAsUser
+  ## @param master.containerSecurityContext.runAsNonRoot Set master-elegible containers' Security Context runAsNonRoot
+  ##
+  containerSecurityContext:
```

Comment:
Missing other important sections like capabilities.drop. See this example from APISIX

```
  ## @param controlPlane.podSecurityContext.seccompProfile.type Set APISIX container's Security Context seccomp profile
  ##
  podSecurityContext:
    enabled: true
    fsGroup: 1001
    seccompProfile:
      type: RuntimeDefault
  ## Configure Container Security Context
  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
  ## @param controlPlane.containerSecurityContext.enabled Enabled APISIX containers' Security Context
  ## @param controlPlane.containerSecurityContext.runAsUser Set APISIX containers' Security Context runAsUser
  ## @param controlPlane.containerSecurityContext.runAsNonRoot Set APISIX containers' Security Context runAsNonRoot
  ## @param controlPlane.containerSecurityContext.readOnlyRootFilesystem Set APISIX containers' Security Context runAsNonRoot
  ## @param controlPlane.containerSecurityContext.allowPrivilegeEscalation Set APISIX container's privilege escalation
  ## @param controlPlane.containerSecurityContext.capabilities.drop Set APISIX container's Security Context runAsNonRoot
  ##
  containerSecurityContext:
    enabled: true
    runAsUser: 1001
    runAsNonRoot: true
    readOnlyRootFilesystem: true
    allowPrivilegeEscalation: false
    capabilities:
      drop: ["ALL"]

```

Reply:
Added [555e0c9](https://github.com/bitnami/charts/pull/17509/commits/555e0c9e237ac6ce10edf3066213e5a673375568)